tests/test_perf_tracking.py

```diff
@@ -183,24 +189,18 @@ def calculate_results(sim_params,
     splits = splits or {}
     commissions = commissions or {}
 
+    adjustment_reader = adjustment_reader or NullAdjustmentReader()
+
     data_portal = create_data_portal_from_trade_history(
         env,
         tempdir,
         sim_params,
-        trade_events
+        trade_events,
     )
+    data_portal._adjustment_reader = adjustment_reader
```

Comment:
Yes, we will. Going to do another round of tightening up the interfaces after getting this in and able to work from a base where all tests pass.